setup.py

```diff
@@ -17,8 +17,7 @@
 from setuptools import setup, find_packages
 
 packages = find_packages()
-module = packages[0]
-# print('module = {}'.format(module))
+module = "ipyaggrid"
```

Comment:
One last comment about this. I think it's find Mario merge it with this, but sometimes small (unrelated) changes can be food for discussion and drag on PR for longer than needed. Especially an important PR like this we want to get in fast. Try to limit your PR's to a single relevant change, thanks 🙏